src/mks/check.py

```diff
@@ -21,9 +21,11 @@ def get_short_status(clients):
             logging.error(ex)
             status = "fail"
 
+        endpoint = client['endpoint'] if client.get('endpoint') else "app/version"
```

Comment:
it will be a good idea to add 
`default_version_endpoint: "app/version"` to the top of config yaml and use it here

Reply:
Sure, then we have to pass the `default_version_endpoint` as the second argument into the `get_short_status()` and `get_long_status()` functions as I made it in the following commit.